core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -1285,7 +1285,6 @@ ExWorkProcRetcode ExHbaseAccessBulkLoadPrepSQTcb::work()
       matches_ = 0;
       currRowNum_ = 0;
       numRetries_ = 0;
-      hFileParamsInitialized_ = FALSE;
```

Comment:
Is this parameter used to create sampling of the loaded data. If so, can you please consider using another condition too.

Also, this flag is being reset to false in ALL_DONE/DONE step. So, commenting out in NOT_STARTED step alone wouldn't help.

I also see this comment in the constructor. Do you think should we retain this comment still.

 hFileParamsInitialized_ = false;  ////temporary-- need better mechanism later